cc3d/player5/Configuration_settings/instructions.txt

```diff
@@ -9,6 +9,10 @@ insert into settings(name, type, value) VALUES("RestartOutputEnable","bool",0);
 insert into settings(name, type, value) VALUES("RestartOutputFrequency","int",100);
 insert into settings(name, type, value) VALUES("RestartAllowMultipleSnapshots","bool",0);
 
+
+insert into settings(name, type, value) VALUES("DemosPath","str","");
+insert into settings(name, type, value) VALUES("LastDemoEditTime","float",1713634361.173719);
```

Comment:
Change to 0 per https://github.com/CompuCell3D/CompuCell3D/pull/716#issuecomment-2081469914